task_2/andrii.lukianchykov/task_2/task_2/task_2/src/main/java/com/bookshop/bookdatabase/BookDataBase.java

```diff
@@ -0,0 +1,85 @@
+package com.bookshop.bookdatabase;
+
+import com.bookshop.book.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.System.out;
+
+public class BookDataBase {
+
+    private Book book;
+    private static List<Book> listOfBooks = new ArrayList<>();
+
+    public BookDataBase() {}
+
+    public BookDataBase(List<Book> listOfBooks) {
+        this.listOfBooks = listOfBooks;
+
+    }
+    public  BookDataBase(Book book) {
```

Comment:
I don't see a reason for that constructor and for storing a single book in a database of books. Your List<Book> is enough for this class.

Reply:
Yeah, i have already created that thing, that can save my books